.github/workflows/component-owners.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Component Owners'
+on:
+  pull_request_target:
+
+jobs:
+  run_self:
+    runs-on: ubuntu-latest
+    name: Auto Assign Owners
+    steps:
+      - uses: dyladan/component-owners@main
```

Comment:
Do you think we should pin it to a sha or tag?

Reply:
I considered it, but this is what Dan has in OTel so I went this way.

Reply:
I'd also be comfortable with slightly less mutability, or at least targeted mutability (like a tag).

Or reducing the permissions for these jobs specifically to what they need to decrease the supply chain risk.
https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
Good points, see [here](https://github.com/open-feature/js-sdk-contrib/pull/446/files#r1259750922).